fem/fe.hpp

```diff
@@ -251,50 +251,48 @@ class FiniteElement
 
 public:
    /// Enumeration for RangeType and DerivRangeType
-   enum { SCALAR, VECTOR };
+   enum RangeT { SCALAR, VECTOR };
```

Comment:
The other enumerations in MFEM spell out the work "Type".  In fact, the only abbreviation I noticed in enumeration names was "Eval" (and other long words like "Derivative").  Perhaps it would be more consistent to use "RangeType", "MapType", and "DerivType".

Reply:
Unfortunately RangeType, MapType, and DerivType are the variable names that use these enum. Do you think we should change the variables to something else?

Reply:
In most cases we use `snake_case` for member variables, so switching to variables like `range_type` makes sense to me.

One potential issue for backward-compatibility is if someone is using a custom class derived from `FiniteElement` which works directly with these protected variables. However, this seems very unlikely.

Reply:
Ok I snake_cased the variables of basic types in that class.